src/Controller/RegisterController.php

```diff
@@ -77,7 +66,8 @@ public function register(): TrustedRedirectResponse {
    */
   public function getRegisterUrl(): Url {
     $config = $this->configFactory->get('oe_authentication.settings');
-    $base_url = $this->casHelper->getServerBaseUrl();
+    $casServerConfig = CasServerConfig::createFromModuleConfig($this->configFactory->get('cas.settings'));
+    $base_url = $casServerConfig->getServerBaseUrl();
```

Comment:
Do we have big advantages by dropping usage of CasHelper service?

Reply:
At this moment this service (casHelper) does not have the "getServerBaseUrl".